package.json

```diff
@@ -87,13 +86,13 @@
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-prettier": "4.0.0",
     "eslint-plugin-vue": "8.3.0",
-    "jest": "27.4.7",
+    "jsdom": "19.0.0",
```

Comment:
Just an idea: what about using [happy-dom](https://github.com/capricorn86/happy-dom)? It should be smaller and maybe a bit faster. [[Reference](https://vitest.dev/config/#environment)]